src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,158 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.MyValidatorException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int MIN_COUNT_OF_CHAR = 6;
+    private static final int MIN_AGE = 18;
+    private static final String LOGIN_EXIST_USER = "ExistLogin";
+    private static final String LOGIN_DEFAULT_USER = "MercedesCar";
+    private static final String PASSWORD = "Password";
+    private static final int USER_AGE = 25;
+    private RegistrationServiceImpl registrationService;
+    private User existUser;
+    private User defaultUser;
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        existUser = new User();
+        existUser.setLogin(LOGIN_EXIST_USER);
+        existUser.setPassword(PASSWORD);
+        existUser.setAge(USER_AGE);
+
+        defaultUser = new User();
+        defaultUser.setLogin(LOGIN_DEFAULT_USER);
+        defaultUser.setPassword(PASSWORD);
+        defaultUser.setAge(USER_AGE);
```

Comment:
You can create Constructor in user and initialize from constructor, try not use redundant line separation

Reply:
as for the empty line, in principle, I will remove it, but I wanted to separate the initialization of the fields of two users, and in general, for me, it is better to initialize these fields in the @BeforeEach block